src/pages/Detail/components/DetailArticle/DetailArticle.jsx

```diff
@@ -1,32 +1,35 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+
+import useFetch from 'pages/Detail/hooks/useFetch';
 import DetailSlider from './DetailSlider';
+import DetailInformation from './DetailInformation';
 
-export default function DetailArticle() {
-  const [detailData, setDetailData] = useState({});
-  const [isLoading, setIsLoading] = useState(true);
+const BASE_URL = {
+  url: 'http://localhost:3000/data/DetailMockData.json',
+};
 
-  useEffect(() => {
-    fetch('http://localhost:3000/data/DetailMockData.json')
-      .then(res => res.json())
-      .then(data => setDetailData(data[0]))
-      .then(setIsLoading(false));
-  }, []);
+export default function DetailArticle() {
+  const { data: data, loading: isLoading } = useFetch({
+    url: `${BASE_URL.url}`,
```

Comment:
```suggestion
    url: BASE_URL,
```
이미 문자열이기 때문에 템플릿 리터럴로 다시 감싸줄 필요 없을거 같아요

Reply:
넵 확인했습니당